src/neptune/internal/backends/neptune_backend.py

```diff
@@ -300,8 +299,8 @@ def fetch_atom_attribute_values(
     def search_leaderboard_entries(
         self,
         project_id: UniqueId,
-        types: Optional[Iterable[ContainerType]] = None,
+        types: Optional[List[ContainerType]] = None,
```

Comment:
Why do we have to update typings here?

Reply:
to avoid a `Iterable` in the future

Reply:
We're not chaining here so what will go wrong here?